### ISSUE TITLE
Impersonate user utility

### ISSUE THREAD

**Issue by sjones4** [Corymbia/eucalyptus#84]
For cloud administration it is sometimes necessary to perform actions against resources in a users account. For the IAM service we have extended the API to allow administrators to pass the account they want to operate on. This approach requires every action/message to have special handling and would require significant effort to extend to all services.

We have a cloud administrator STS action that returns credentials for impersonating an account/user and we could add a command line tool with a workflow similar to clcadmin-assume-system-credentials e.g.

```
# eval $(clcadmin-impersonate-user someaccountname)
#
# euca-describe-instances
#
# # perform other euca2ools actions as the impersonated account
```

In addition to resource management, this could be used to test policies / quotas for accounts without the need to create credentials.

[EUCA-12370](https://eucalyptus.github.io/eucalyptus-jira/generated/euca/EUCA-12370.html)

**Comment by sjones4** [comment]
Fixed by Corymbia/eucalyptus@23828d5f8b50c1b4dadf24555a6db67de5849169 Corymbia/eucalyptus@de22bc41840a591189ce9c26889b726c1a3ca75e

**Comment by sjones4** [comment]
`clcadmin-impersonate-user` and `clcadmin-release-credentials` added in master.
